stable/admin/templates/_helpers.tpl

```diff
@@ -144,3 +144,40 @@ Import ENV vars from configMaps
 {{- define "admin.envFrom" }}
 envFrom: [{{- range $index, $map := .Values.admin.envFrom.configMapRef }}{{if gt $index 0}},{{end}} configMapRef: { name: {{$map}} } {{ end }}]
 {{- end -}}
+
+{{/*
+Define the cluster domains
+*/}}
+{{- define "cluster.domain" -}}
+{{- .Values.cloud.cluster.domain | default "cluster.local" }}
```

Comment:
I'm not sure what the syntax is for this template language, but I see both of the following used interchangeably:
```
{{- ... -}}
```

```
{{ ... }}
```

This one has `{{- ... }}`. Is this a typo? Are all of these valid?

Reply:
Dirk Butson just explained it to me; `{{-` means strip whitespace before and `-}}` means strip whitespace after.